src/library_websocket.js

```diff
@@ -109,6 +109,7 @@ var LibraryWebSocket = {
 #endif
       return {{{ cDefine('EMSCRIPTEN_RESULT_INVALID_TARGET') }}};
     }
+    HEAPU32[urlLength>>2] = lengthBytesUTF8(socket.url);
```

Comment:
is this change related to this PR? It didn't seem like the PR was focused on fixing library_websocket?

Reply:
Hmm no that's a separate fix that I did while authoring this.

Reply:
Please separate it into another PR then - I worry about confusion in bisection if we end up with a regression here.

Reply:
Landed that as https://github.com/emscripten-core/emscripten/pull/9896